s3-sanity-test.sh

```diff
@@ -58,11 +58,13 @@ then
     ldappasswd=$ldap_admin_pwd
 fi
 
-USAGE="USAGE: bash $(basename "$0") [--help]
+USAGE="USAGE: bash $(basename "$0")
 Run S3 sanity test.
 where:
-    --help      display this help and exit
-    --clean     clean s3 resources if present
+    -h     display this help and exit
+    -c     clean s3 resources if present
+    -e     specify the server endpoint.it should be localhost(127.0.0.1)
```

Comment:
specify the s3 server endpoint, pass localhost if running on s3 server node

Reply:
done